server/schedulers/hot_region.go

```diff
@@ -538,3 +603,36 @@ func (h *balanceHotRegionsScheduler) GetStoresScore() map[uint64]float64 {
 	}
 	return storesScore
 }
+
+func calcPendingWeight(op *operator.Operator) (weight float64, remove bool) {
+	if op.CheckExpired() || op.CheckTimeout() {
+		return 0, true
+	}
+	status := op.Status()
+	if !operator.IsEndStatus(status) {
+		return 1, false
```

Comment:
why 1?

Reply:
The operator now is running, so we will estimate that all influence of this region is not observed by statistics.